src/huggingface_hub/_space_api.py

```diff
@@ -118,3 +121,31 @@ def __init__(self, data: Dict) -> None:
         self.sleep_time = data["gcTimeout"]
         self.storage = data["storage"]
         self.raw = data
+
+
+@dataclass
+class SpaceVariable:
+    """
+    Contains information about the current variables of a Space.
+
+    Args:
+        key (`str`):
+            Variable key. Example: `"MODEL_REPO_ID"`
+        value (`str`):
+            Variable value. Example: `"the_model_repo_id"`.
+        description (`str` or None):
+            Description of the variable. Example: `"Model Repo ID of the implemented model"`.
+        updatedAt (`datetime`):
+            datetime of the last update of the variable.
+    """
+
+    key: str
+    value: str
+    description: Optional[str]
+    updatedAt: datetime
```

Comment:
```suggestion
    updated_at: datetime
```

For consistency, let's use `snake_case` for all attributes